src/main/scala/cromwell/binding/values/WdlArray.scala

```diff
@@ -10,5 +10,13 @@ case class WdlArray(wdlType: WdlArrayType, value: Seq[WdlValue]) extends WdlValu
   if (typesUsedInValue.size > 1) {
     throw new UnsupportedOperationException(s"Cannot construct array with a mixed types: $value")
   }
-  override def toWdlString(): String = s"[${value.map{x=>x.toWdlString}.mkString(", ")}]"
+
+  override def toWdlString(): String = s"[${value.map { x => x.toWdlString }.mkString(", ")}]"
```

Comment:
IntelliJ doesn't like that the base is `def toWdlString:`, and this overrides as the empty-params `override def toWdlString()`. For consistency I'd remove the `()`.

And another `.map{}`. Can this one be `.map()`?

Or revert the line altogether, and leave the blame on the sins of PRs past.